packages/replay-next/src/suspense/SourcesCache.ts

```diff
@@ -187,6 +188,18 @@ export const sourcesByIdCache = createSingleEntryCache<
   },
 });
 
+export const sourcesByPartialUrlCache: Cache<
```

Comment:
Kind of a nit: This cache feels a bit unnecessary. Seems like we could `useMemo` or in some other way cache the result of this filter action without adding a whole new cache for it. We're really only using it to look for a single file anyway (react-dom).

Reply:
It's being used for `react-dom`, but the call is inside of a couple different caches, and I've got a spot in the follow-up PR that I just noticed ought to be using this for Redux files as well.